### DISCUSSION TITLE
[docs] guides/async is not listed  in the guide section of the left navigation bar of the document

### DISCUSSION THREAD

**Discussion by maple5233** [pmndrs/jotai#1871] [Q&A] [answered]
## Summary

https://jotai.org/docs/guides/async should be listed in the guide section of the left navigation bar of the document, I guess. 😄

<img width="252" alt="image" src="https://user-images.githubusercontent.com/16346984/220842682-ff43515d-8b2c-4b70-adc0-6d88f0285431.png">

## Link to reproduction

https://jotai.org/docs/introduction

**Comment by dai-shi** [comment]
@sandren Is this intentional or?

**Reply by dai-shi** [reply, marked as answer]
I think this is intentional for now, but thanks for the feedback.
